libraries/GCS_MAVLink/GCS_Common.cpp

```diff
@@ -1060,6 +1060,7 @@ ap_message GCS_MAVLINK::mavlink_id_to_ap_message_id(const uint32_t mavlink_id) c
         { MAVLINK_MSG_ID_CAMERA_SETTINGS,       MSG_CAMERA_SETTINGS},
         { MAVLINK_MSG_ID_CAMERA_FOV_STATUS,     MSG_CAMERA_FOV_STATUS},
         { MAVLINK_MSG_ID_CAMERA_CAPTURE_STATUS, MSG_CAMERA_CAPTURE_STATUS},
+        { MAVLINK_MSG_ID_CAMERA_THERMAL_RANGE,  MSG_CAMERA_THERMAL_RANGE},
```

Comment:
```suggestion
#if AP_CAMERA_SEND_THERMAL_RANGE_ENABLED
        { MAVLINK_MSG_ID_CAMERA_THERMAL_RANGE,  MSG_CAMERA_THERMAL_RANGE},
#endif
```

Reply:
I might leave this one out if that's OK.  If you look up a few lines you'll see that the CAMERA_FOV_STATUS is also not behind a definition.

Reply:
This one really should be in.  Feel free to fix the other one - and group them together would be even nicer!

Reply:
ok, added, thanks!